cliquet/resource/__init__.py

```diff
@@ -1046,6 +1088,17 @@ def get_parent_id(self, request):
         """
         return ''
 
+    def _extract_fields(self):
+        """Override default fields extraction from QueryString to allow
+        the permission_field to exist.
+        """
+
+        fields = super(ShareableResource, self)._extract_fields()
+        if fields:
+            fields.append(self.model.permissions_field)
+
+        return fields
```

Comment:
Permissions for each records, this would be useful to be able to sync permissions between two servers at once.
See https://github.com/Kinto/kinto.py/pull/26/files#diff-7d388f51c9ff65051463565ec3207ea8R32